Wikipedia/Code/FontSizeSliderViewController.swift

```diff
@@ -32,7 +32,7 @@ class FontSizeSliderViewController: UIViewController {
         
         slider.isAccessibilityElement = true
         slider.accessibilityTraits = UIAccessibilityTraitAdjustable
-        slider.accessibilityLabel = WMFLocalizedString("reading-themes-controls-accessibility-text-size-slider", value: "Text size slider", comment: "Accessibility label for the text size slider in the Appearance Settings\n{{Identical|Text size slider}}")
+        slider.accessibilityLabel = WMFLocalizedString("appearance-settings-accessibility-text-size-slider", value: "Text size slider", comment: "Accessibility label for the text size slider in the Appearance Settings\n{{Identical|Text size slider}}")
```

Comment:
This can be consolidated in `CommonStrings` - use `reading-themes-controls-accessibility-text-size-slider` since we have translations for that